k-distribution/src/main/scripts/lib/pyk/kast.py

```diff
@@ -114,9 +114,13 @@ def let_att(self: W, att: KAtt) -> W:
         ...
 
     def map_att(self: W, f: Callable[[KAtt], KAtt]) -> W:
+        if self.att is None:
+            return self
         return self.let_att(att=f(self.att))
 
     def update_atts(self: W, atts: Mapping[str, Any]) -> W:
+        if self.att is None:
+            return self.let_att(att=KAtt.of(**atts))
```

Comment:
For a `WithAtt` instance `att` should never be `None` . If there are such failures, we should root-cause them.